src/components/Card.vue

```diff
@@ -61,9 +61,19 @@ export default {
     return {
       editing: null,
       assigningUser: false,
+      assignableUsers: [],
     };
   },
   methods: {
+    async switchAssigning() {
+      if (!this.assigningUser) {
+        this.card.column.board = await axios.get(`/board/${this.$route.params.id}`)
+          .then((response) => response.data);
```

Comment:
je kunt ook in de .then `this.card.column.board = response.data` zetten, dan is er geen await nodig

Reply:
dat is dan ook hetzelfde als alle andere axios request, nu is het hier anders

Reply:
Vind deze manier eigenlijk een stuk mooier maar je hebt wel gelijk met de consistencie